### PR TITLE
Show available keys for chargeback statement csv export

### PR DESCRIPTION
**Pull request by felixzieger**

### CODE REVIEW

File: docs/meshstack.billing.md [hunk 1]
```diff
@@ -239,7 +239,31 @@ The following configuration options are available at `mesh.kraken.productcatalog
...
+### Available metadata keys
+
+The following metadata keys are derived from meshStack metadata and therefore available on every meshStack implementation
+
+| Key                   | Description                                      |
+|-----------------------|:-------------------------------------------------|
+| tenantLandingZone     | Name of the Landing Zone used by the Tenant      |
+| last_modified         | Date when metadata was last modified             |
+| contactMail           | email address of the project owner               |
+| ownerUsername         | Username of the project owner                    |
```
Comment (chewbeccy) [inline docs/meshstack.billing.md:253]: I think the "owner" is confusing as we will introduce the customer owner.

File: docs/meshstack.billing.md [hunk 2]
```diff
@@ -239,7 +239,31 @@ The following configuration options are available at `mesh.kraken.productcatalog
 
 Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
 
-The attributes that shall be part of the billing info on the chargeback statements can be configured as follows.
+The attributes that shall be part of the billing info on the chargeback statements can be configured.
+
+### Available metadata keys
```
Comment (chewbeccy) [inline docs/meshstack.billing.md:244]: Isn't this very customer specific? The metadata available is very specific to a meshStack

Comment (felixzieger) [reply docs/meshstack.billing.md:244]: The meshStack specific part is under `Available tag keys`

File: docs/meshstack.billing.md (hunk 1 above)
Comment (felixzieger) [reply docs/meshstack.billing.md:253]: The `ownerUsername` tag is available on every meshStack. When a customer owner is introduced, this should be changed accordingly in this document.

**Review by chewbeccy — APPROVED** [review]

**Pull request merged**